### PR TITLE
feat: new loading and regenerating state.

### PR DESCRIPTION
**Pull request by taylorgp**

### CODE REVIEW

File: app/(candidate)/dashboard/content/[slug]/components/ContentEditor.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { globalSnackbarState } from '@shared/utils/Snackbar';
 import { updateCampaign } from 'app/(candidate)/onboarding/shared/ajaxActions';
 import PlanVersion from './PlanVersion';
 import PrimaryButton from '@shared/buttons/PrimaryButton';
-import LoadingAI from 'app/(candidate)/onboarding/[slug]/campaign-plan/components/LoadingAI';
```
Comment (tomer-tgp) [inline app/(candidate)/dashboard/content/[slug]/components/ContentEditor.js:9]: if this is not used, let's delete the file

**Review by tomer-tgp — CHANGES_REQUESTED** [review]
Small comment. besides that looks good

**Comment by taylorgp** [comment]
I believe it is still used by the onboarding ai content.

**Comment by taylorgp** [comment]
this should be good to merge

**Review by tomer-tgp — APPROVED** [review]

**Pull request merged**